components/applications-service/pkg/storage/postgres/schema/sql/03_service_group_view.up.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE service ADD COLUMN full_pkg_ident TEXT NOT NULL DEFAULT '';
+
+CREATE OR REPLACE VIEW service_groups AS
```

Comment:
The name of this file doesn't match with the view name: 

- File singular: `03_service_group_view.up.sql`
- View plural: `VIEW service_groups`

More in the **naming** topic, _I would suggest calling this view differently_ since I have a feeling that it will be confusing in the near future, currently we have a table called `service_group` and this view is created as `service_groups`, if I am new and look at them both, I wouldn't be able to differentiate them. 

- To me, the `service_group` table is the actual definition of a service-group.
- Now, I think the new view is **an aggregation of the service-groups health**, such information is combined from all its services, deployment id, application, and environment name, etc. 

I would suggest this view to be called: `service_group_health_aggregation`. (or something similar)

Reply:
I agree with @afiune the naming is confusing. Just by looking at the names of the 'service_groups' view and 'service_group' table I would be confused. It looks like the view is the one that needs the rename.